src/tree.rs

```diff
@@ -958,19 +958,30 @@ impl Tree {
     checkout: CheckoutType,
     detach: bool,
     fetch_tags: bool,
+    pull_lfs: bool,
+    repo_compat: bool,
   ) -> Result<i32, Error> {
     let mut ssh_masters = HashMap::new();
-    let result = self.sync_impl(
-      config,
+    let mut result = self.sync_impl(
+      config.clone(),
       pool,
-      sync_under,
+      sync_under.clone(),
       fetch_type,
       fetch_target,
       checkout,
       detach,
       fetch_tags,
       &mut ssh_masters,
     );
+    if pull_lfs {
+      result = self.forall(
+        config,
+        pool,
+        sync_under,
+        "test -e .lfsconfig && git lfs pull && git lfs prune",
```

Comment:
Do we actually want to `git lfs prune` here? I think that should be something that you have to explicitly ask for.

(Also, it'll be significantly faster to test for .lfsconfig from rust instead of spawning a shell to do it, but that can wait until we make this automatic)